planb/cli.py

```diff
@@ -144,6 +144,7 @@ def extend(from_region: str,
 @click.option('--odd-host', '-O', type=str, required=True)
 @click.option('--region', type=str, required=True)
 @click.option('--force-termination', is_flag=True, default=False)
+@click.option('--no-prompt', is_flag=True)
```

Comment:
Could we specify the default explicitly?

Reply:
Maybe, but I'm not sure we should.  I believe the semantics of flag options is `False` if not specified.  Also, we didn't do that in other places where we used `is_flag=True`, e.g. see `remote` command below.